scene/register_scene_types.cpp

```diff
@@ -765,7 +765,7 @@ void register_scene_types() {
 	GDREGISTER_CLASS(RibbonTrailMesh);
 	GDREGISTER_CLASS(PointMesh);
 	GDREGISTER_VIRTUAL_CLASS(Material);
-	GDREGISTER_VIRTUAL_CLASS(BaseMaterial3D);
+	GDREGISTER_ABSTRACT_CLASS(BaseMaterial3D);
```

Comment:
Could maybe make sense as virtual? Either way, all this can be discussed in follow-ups if/when the needs arise and then changing it is just a macro rename away.